subprojects/docs/src/samples/userguide/dependencyManagement/artifactTransforms/minify/groovy/build.gradle

```diff
@@ -142,7 +142,7 @@ repositories {
 // tag::artifact-transform-attribute-setup[]
 
 dependencies {                                 // <4>
-    implementation('com.google.guava:guava:27.1-jre')
+    implementation('com.google.guava:guava:28.0-jre')
```

Comment:
This is the same case as with Groovy: we shouldn't upgrade samples, instead we should figure out a way to do this automatically.

Reply:
reverted